deployments/helm/KubeArmorOperator/kubearmor-operator.yaml

```diff
@@ -0,0 +1,333 @@
+---
```

Comment:
Have we added this manifest generated by `helm template` intentionally? :thinking: 
We already have the one generated by kubebuilder for testing purposes kept here - https://github.com/kubearmor/KubeArmor/tree/main/deployments/operator